bench/tiered/push_pull.c

```diff
@@ -204,6 +210,69 @@ fill_random_data(void)
     data_str[str_len - 1] = '\0';
 }
 
+/*
+ * remove_local_cached_files --
+ *     Remove local cached files and cached folders.
+ */
+static void
+remove_local_cached_files(const char *home)
+{
+    struct dirent *dir_entry;
+    DIR *dir;
+
+    char *tablename;
+    char delete_file[512], file_prefix[1024], rm_cmd[512];
+    int highest, index, nmatches, objnum, status, success_status;
+
+    nmatches = highest = objnum = success_status = 0;
+    tablename = opts->uri;
+
+    if (!WT_PREFIX_SKIP(tablename, "table:"))
+        testutil_die(EINVAL, "unexpected uri: %s", opts->uri);
+
+    testutil_check(__wt_snprintf(file_prefix, sizeof(file_prefix), "%s-000", tablename));
+
+    dir = opendir(home);
+    testutil_assert(dir != NULL);
+
+    while ((dir_entry = readdir(dir)) != NULL) {
+        if (strcmp(dir_entry->d_name, ".") == 0 || strcmp(dir_entry->d_name, "..") == 0)
+            continue;
+
+        if (!WT_PREFIX_MATCH(dir_entry->d_name, tablename))
+            continue;
+
+        ++nmatches;
+
+        sscanf(dir_entry->d_name, "%*[^0-9]%d", &objnum);
+        highest = WT_MAX(highest, objnum);
+    }
+
+    closedir(dir);
+
+    if (highest > 1 && nmatches > 1) {
+        for (index = 1; index < highest; index++) {
+            testutil_check(__wt_snprintf(
+              delete_file, sizeof(delete_file), "rm -f %s/%s*0%d.wtobj", home, file_prefix, index));
+            status = system(delete_file);
+
+            /* Count the number of times the files are deleted. */
+            if (status == 0)
+                ++success_status;
+
+            if (status < 0)
+                testutil_die(status, "system: %s", delete_file);
```

Comment:
This might be easier to understand like this:
if (status != 0)
    testutil_die(...)

I think checking any non-zero status means we don't need the assert below and then we don't need `success_status` and it simplifies the code.

Reply:
Fix in 4eb0e1bc7e8a9c3651cc9be94d4f7e11f2520ac5